labgrid/util/ssh.py

```diff
@@ -124,6 +124,7 @@ class SSHConnection:
         validator=attr.validators.instance_of(str)
     )
     _forwards = attr.ib(init=False, default=attr.Factory(dict))
+    stderr_merge = attr.ib(default=False, validator=attr.validators.instance_of(bool))
```

Comment:
It seems this should be an option for the actual run command, as we always create separate ssh sessions (in the same shared connection).

Reply:
Changed.